.github/workflows/publishQA.yml

```diff
@@ -0,0 +1,61 @@
+# On merge to qa, build a container and deploy to ECR
+name: Publish QA
+on:
+  pull_request:
+    branches: [qa]
+    types: [closed]
+
+jobs:
+  publish_and_deploy_qa:
+    name: Publish image to ECR
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        project: [BPL, OE, SE]
+        stage: [cm_scripts_active, cm_webapp_active, cm_exec_active]
```

Comment:
Do we push `cm_exec_active` to ECR? I thought it was just scripts and webapp

Reply:
I checked ECR and you're right, we don't push exec. I'll remove that.